apps/mobile/src/components/EventsAndClubsScreen/components/ClubsScreen/components/ClubsList.tsx

```diff
@@ -72,6 +83,8 @@ export function ClubsList(): JSX.Element {
       ItemSeparatorComponent={Separator}
       renderItem={renderItem}
       keyExtractor={atomKeyExtractor}
+      onRefresh={() => Effect.runFork(syncAllClubsHandleStateWhenNotFound())}
```

Comment:
Make sure to test on iOS, the progress spinner is not visible. This prop needs to be set: 

```
indicatorStyle="white"
refreshControl={
        <RefreshControl
          refreshing={refreshing ?? false}
          onRefresh={onRefresh ?? (() => {})}
          tintColor={getTokens().color.greyAccent5.val}
        />
      }
```